tests/plugins/test_koji.py

```diff
@@ -83,6 +83,9 @@ def getRepo(self, repo):
             return None
         return GET_REPO_RESPONSE
 
+    def ssl_login(self, cert, ca, serverca, proxyuser=None):
+        return True
```

Comment:
I'll update the test to make sure `ca`, `cert` and `serverca` are checked, but its pretty complicated, as koji session can't store test vars in `workflow` - and workflow can't reach the particular instance of the plugin